package.json

```diff
@@ -20,19 +20,17 @@
     "client library"
   ],
   "dependencies": {
+    "axios": "^0.17.1",
     "gtoken": "^1.2.3",
     "jws": "^3.1.4",
-    "lodash.isstring": "^4.0.1",
-    "lodash.merge": "^4.6.0",
-    "request": "^2.81.0"
+    "lodash.isstring": "^4.0.1"
   },
   "devDependencies": {
+    "@types/jws": "^3.1.0",
```

Comment:
qq: I don't see an import of `@types/axios`. Intentional?

Reply:
Yup - they kindly package their types in the npm module :)

Reply:
I am curious how.. I don't see a `types` property in their [`package.json`](https://github.com/axios/axios/blob/master/package.json)

Reply:
It has `typings` and `types` and `typings` are synonymous: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html